src/config/dynamic-theme-fixes.config

```diff
@@ -265,6 +265,20 @@ CSS
 
 ================================
 
+alipay.com
+
+INVERT
+.alipay-logo
+#J_logoHomeUrl
+.global-logo
+
+CSS
+.qrcode-detail-img {
+    background-color: white !important;
```

Comment:
Replace `white` with `${black}`

Reply:
The QR code image itself is always black on white regardless of our theme, so I explicitly used `white` here instead of `${black}` and it works correctly. Using `${black}` here breaks it on Light theme.